### ISSUE TITLE
about model ，about cifa10

### ISSUE THREAD

**Issue by Hephaestusxg** [G-U-N/PyCIL#83]
Sorry to bother yo, is saving the weights done in the after task function? I'd like to ask for some guidance on this. However, I noticed that when using DER, this function is not utilized. Can I save the model weights at the initial stage as well as after each incremental step? For example, the weights for 8 classes, 9 classes, and 10 classes (on CIFAR-10).

**Comment by zhoudw-zdw** [comment]
See https://github.com/G-U-N/PyCIL/issues/77#issuecomment-2098657237

**Comment by Hephaestusxg** [comment]
> See [#77 (comment)](https://github.com/G-U-N/PyCIL/issues/77#issuecomment-2098657

> See [#77 (comment)](https://github.com/G-U-N/PyCIL/issues/77#issuecomment-2098657237)
Thank you for your response. When I printed the model structure, I noticed that the fully connected (fc) layer is set for 10-class classification for CIFAR-10. My previous understanding was that during the initial training stage, for instance, if there are initially 7 classes, the fc layer should be set for 7-class classification. Based on what the code shows, the accuracy calculated this way would be inaccurate, right?
